internal/utils/csi.go

```diff
@@ -119,6 +121,27 @@ var EtcSelinuxVolume = corev1.Volume{
 	},
 }
 
+func LogDirVolume(LogHostPath, CsiComponentName string) corev1.Volume {
+	return corev1.Volume{
+		Name: csiLogDirectoryVolumeName,
+		VolumeSource: corev1.VolumeSource{
+			HostPath: &corev1.HostPathVolumeSource{
+				Path: fmt.Sprintf("%s/log/%s", LogHostPath, CsiComponentName),
+				Type: ptr.To(corev1.HostPathDirectoryOrCreate),
+			},
+		},
+	}
+}
+
+func LogSidecarDirVolume(LogHostPath, CsiComponentName string) corev1.Volume {
+	return corev1.Volume{
+		Name: csiSidecarLogDirectoryVolumeName,
+		VolumeSource: corev1.VolumeSource{
+			EmptyDir: &corev1.EmptyDirVolumeSource{},
+		},
+	}
+}
+
```

Comment:
Please move these to after the definitions of the existing volume functions

Reply:
I see the other volume functions are just below it